src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Loader } from '../components/Loader';
+import { PeopleList } from '../components/PeopleList';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { getPeople } from '../api';
+import { Person } from '../types/Person';
+import { SortType, checkQuery } from '../utils/sortHelpers';
+
+const getParent = (personList: Person[], parentName: string | null) => {
+  return personList.find(({ name }) => name === parentName);
+};
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setisError] = useState(false);
+
+  const [searchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const chosenSex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
```

Comment:
move all searchParams names to some enum and reuse them everywhere, it will help us to easily change some name